"make proto" is broken

**Is this a BUG REPORT or FEATURE REQUEST?**:
BUG

**What happened**:
Running `make proto` throws an error even if I have all the tools

**What you expected to happen**:
`make proto` shouldn't throw an error

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:
Output of the command:
```
# make proto
Generating protobuf definitions from api/api.proto
protoc -I /home/disrani/go/src/github.com/libopenstorage/openstorage \
	-I /usr/local/include \
	-I /home/disrani/go/src/github.com/grpc-ecosystem/grpc-gateway/third_party/googleapis \
	--go_out=plugins=grpc:. \
	/home/disrani/go/src/github.com/libopenstorage/openstorage/api/api.proto
protoc -I /home/disrani/go/src/github.com/libopenstorage/openstorage \
	-I /usr/local/include \
	-I /home/disrani/go/src/github.com/grpc-ecosystem/grpc-gateway/third_party/googleapis \
	--grpc-gateway_out=logtostderr=true:. \
	/home/disrani/go/src/github.com/libopenstorage/openstorage/api/api.proto
protoc -I /home/disrani/go/src/github.com/libopenstorage/openstorage \
	-I /usr/local/include \
	-I /home/disrani/go/src/github.com/grpc-ecosystem/grpc-gateway/third_party/googleapis \
	--swagger_out=logtostderr=true:/home/disrani/go/src/github.com/libopenstorage/openstorage/api/server/sdk \
	/home/disrani/go/src/github.com/libopenstorage/openstorage/api/api.proto
Generating grpc protobuf definitions from pkg/flexvolume/flexvolume.proto
protoc -I/usr/local/include -I/home/disrani/go/src/github.com/libopenstorage/openstorage -I/home/disrani/go/src/github.com/grpc-ecosystem/grpc-gateway/third_party/googleapis --go_out=plugins=grpc:. /home/disrani/go/src/github.com/libopenstorage/openstorage/pkg/flexvolume/flexvolume.proto
protoc -I/usr/local/include -I/home/disrani/go/src/github.com/libopenstorage/openstorage -I/home/disrani/go/src/github.com/grpc-ecosystem/grpc-gateway/third_party/googleapis --grpc-gateway_out=logtostderr=true:. /home/disrani/go/src/github.com/libopenstorage/openstorage/pkg/flexvolume/flexvolume.proto
Generating protobuf definitions from pkg/jsonpb/testing/testing.proto
protoc -I /home/disrani/go/src/github.com/libopenstorage/openstorage /home/disrani/go/src/github.com/libopenstorage/openstorage/pkg/jsonpb/testing/testing.proto --go_out=plugins=grpc:.
Generating gRPC clients
make -C api/client/sdk
make[1]: Entering directory '/home/disrani/go/src/github.com/libopenstorage/openstorage/api/client/sdk'
make -C python
make[2]: Entering directory '/home/disrani/go/src/github.com/libopenstorage/openstorage/api/client/sdk/python'
virtualenv "build"
New python executable in /home/disrani/go/src/github.com/libopenstorage/openstorage/api/client/sdk/python/build/bin/python
Installing setuptools, pip, wheel...done.
bash -c "source "build"/bin/activate && \
pip install grpcio grpcio-tools && \
python -m grpc_tools.protoc \
	-I"../../../../api" \
	-I/home/disrani/go/src/github.com/grpc-ecosystem/grpc-gateway/third_party/googleapis \
	--python_out=. \
	--grpc_python_out=. \
	"../../../../api"/api.proto"
Requirement already satisfied: grpcio in ./build/lib/python2.7/site-packages (1.12.1)
Requirement already satisfied: grpcio-tools in ./build/lib/python2.7/site-packages (1.12.1)
Requirement already satisfied: six>=1.5.2 in ./build/lib/python2.7/site-packages (from grpcio) (1.11.0)
Requirement already satisfied: enum34>=1.0.4 in ./build/lib/python2.7/site-packages (from grpcio) (1.1.6)
Requirement already satisfied: futures>=2.2.0 in ./build/lib/python2.7/site-packages (from grpcio) (3.2.0)
Requirement already satisfied: protobuf>=3.5.0.post1 in ./build/lib/python2.7/site-packages (from grpcio-tools) (3.6.0)
Requirement already satisfied: setuptools in ./build/lib/python2.7/site-packages (from protobuf>=3.5.0.post1->grpcio-tools) (39.2.0)
bash -c "source "build"/bin/activate && \
python -m grpc_tools.protoc \
	-I"../../../../api" \
	-I/home/disrani/go/src/github.com/grpc-ecosystem/grpc-gateway/third_party/googleapis \
	--python_out=. \
	--grpc_python_out=. \
	/home/disrani/go/src/github.com/grpc-ecosystem/grpc-gateway/third_party/googleapis/google/api/annotations.proto"
bash -c "source "build"/bin/activate && \
python -m grpc_tools.protoc \
	-I"../../../../api" \
	-I/home/disrani/go/src/github.com/grpc-ecosystem/grpc-gateway/third_party/googleapis \
	--python_out=. \
	--grpc_python_out=. \
	/home/disrani/go/src/github.com/grpc-ecosystem/grpc-gateway/third_party/googleapis/google/api/http.proto"
make[2]: Leaving directory '/home/disrani/go/src/github.com/libopenstorage/openstorage/api/client/sdk/python'
make -C ruby
make[2]: Entering directory '/home/disrani/go/src/github.com/libopenstorage/openstorage/api/client/sdk/ruby'
make[2]: command: Command not found
Makefile:7: *** "Please install grpc rools for ruby: gem install grpc && gem install grpc-tools".  Stop.
make[2]: Leaving directory '/home/disrani/go/src/github.com/libopenstorage/openstorage/api/client/sdk/ruby'
Makefile:8: recipe for target 'ruby' failed
make[1]: *** [ruby] Error 2
make[1]: Leaving directory '/home/disrani/go/src/github.com/libopenstorage/openstorage/api/client/sdk'
Makefile:101: recipe for target 'proto' failed
make: *** [proto] Error 2
```
**Environment**:
- Container Orchestrator and version:
- Cloud provider or hardware configuration:
- OS (e.g. from /etc/os-release): Ubuntu
- Kernel (e.g. `uname -a`): Linux disrani-thinkpad 4.13.0-43-generic #48~16.04.1-Ubuntu SMP Thu May 17 12:56:46 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
- Install tools:
- Others:

--- a/api/client/sdk/ruby/Makefile
+++ b/api/client/sdk/ruby/Makefile
@@ -1,4 +1,4 @@
-HAS_GRPCTOOLS := $(shell command -v grpc_tools_ruby_protoc)
+HAS_GRPCTOOLS := $(shell command -v grpc_tools_ruby_protoc 2> /dev/null)
 APIDIR := "../../../../api"
 all: grpc
 

